content/02.introduction.md

```diff
@@ -24,69 +24,59 @@ Further, we aim to amplify the true signal out of the vast noise produced by tho
+The innate immune response targets pathogens in circulation.
+This response depends on several types of leukocytes (white blood cells) that are professional phagocytic cells, which means they are able to detect and engulf an antigen, enveloping it in a phagosome.
+The conditions within the phagosome are then modulated to destroy the pathogen through fusion of the phagosome with a lysosome or granule to form a phagolysosome.
+This allows the cell to break the pathogen into its component peptides. 
+Monocytes, which include both dendritic cells and macrophages, are both phagocytic white blood cells and professional antigen-presenting cells, which meaning that they can then present the resulting viral peptides on the cell surface using Major Histocompatability Complex II (MHC-II) proteins.
+However, research in related viruses suggests that SARS-CoV-2 may possess some mechanisms that facilitate evasion of the antibody-mediated response.
+_In vitro_ analysis of nidoviruses including SARS-CoV suggests that these viruses can induce the development of double-membrane vesicles, preventing the dsRNA signature from being detected by the host immune system [@doi:10.1128/jvi.02501-05].
+This protective envelope can therefore insulate these coronaviruses from the innate immune response's detection mechanism [@doi:10.1016/j.jpha.2020.03.001].
+Presentation of the antigen is important because this step allows it to be detected by B-lymphocytes, also called B-cells, which are a type of white blood cell with antibodies bound to the membrane.
+Individual B-cells have unique membrane-bound antibodies.
+A B-cell is able to bind to an antigen only if its antibody is a match.
+For example, toll-like receptor 3 (TLR3) can detect double-stranded RNA (dsRNA) present during viral replication, allowing it to recognize a viral threat.
+The binding of a pattern recognition receptor (PRR), such as a TLR, to the antigen allows the B-cell to engulf it the virus via endocytocis.
+Once the antigen has been processed and presented on the cell surface, helper T-cells can bind to the MHC-II/antigen complex.
+This second binding step catalyzes the release of cytokines as part of the inflammatory response, as well as the proliferation of active B-cells (plasma cells) capable of producing the antibodies that respond to the antigen.
+Coronaviruses such as SARS-CoV are also able to evade the humoral immune response through other mechanisms, such as inhibiting certain cytokine pathways or down-regulating antigen presentation by the cells [@doi:10.1128/jvi.02501-05].
+Additionally, the cell will produce memory B-cells that can retain information about how to respond to future immune challenges by this pathogen.
+Understanding the humoral response to SARS-CoV-2 is important because this component of the immune response works to reduce the severity of infection in the short term and is also the mechanism through which the body can respond to reexposure.
```

Comment:
For lines 58-66, it gets a bit confusing to read because the order of sentences makes it seems like TLR & B-cell immunoglobulin receptors are the same which they're not. Options: (A) clarify the immunology background explanatory sentences (requires making sure to distinguish different receptors, different recognition mechanisms/receptors, different presentation mechanisms/receptors, which cells have which receptors, & which cells recognize which receptors -- we can sort through this or it may be easier to ask a more frequent immunology researcher like @gregszetoAI or @matfax to write this up but that may be too much) or, as the simpler/shortcut option, (B) skip some of the immunology background explanatory sentences & just keep the sentences which focus on the SARS-CoV-2 information, although I understand that at least some (if not all) the immunology background explanatory sentences may be needed/useful for context at least sometimes.

Reply:
@rando2 & rest, *reminder* for us we still need to have an immunologist (or someone looking very carefully at the specific relevant immunology) to check & adjust-if-needed this stuff discussed in the comments above (what is now lines 164-174)